hasher-matcher-actioner/scripts/script_utils.py

```diff
@@ -127,6 +127,30 @@ def send_single_submission_url(
         )
         put_response.raise_for_status()
 
+    def submit_single_url(
```

Comment:
`submit_from_url` instead?

until we batch the `single` is redundant anyway imo